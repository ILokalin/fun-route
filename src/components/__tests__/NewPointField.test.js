import React from 'react';
import ReactTestUtils from 'react-dom/test-utils';
import rendrer from 'react-test-renderer';
import NewPointField from "../NewPointField/NewPointField"
// import faker from 'faker';
// import puppeteer from 'puppeteer';




describe('New point field Component create', () => {
  
// function test() {
//   return true
// }

  it('render snap', () => {
    const component = rendrer.create(<NewPointField
    
                                        placemarkCoord = {[0,0]}
                                        placemarkAddress  = {'Тестовая строка'}/>);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();

    })
})

