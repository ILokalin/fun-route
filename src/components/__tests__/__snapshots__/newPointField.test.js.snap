// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`New point field Component create render snap 1`] = `
<div
  className="new-point"
>
  <h3
    className="new-point__caption"
  >
    Назовите новую точку
  </h3>
  <input
    autoComplete="off"
    className="new-point__input"
    name="address"
    onKeyDown={[Function]}
    placeholder="Тестовая строка"
    title="Добавьте точку в маршрут"
    type="text"
  />
  <button
    className="new-point__add-button"
    onClick={[Function]}
    type="button"
  >
    <svg
      aria-labelledby="title"
      className="icon"
      fill="none"
      viewBox="0 0 30 30"
      xmlns="http://www.w3.org/2000/svg"
    >
      <defs />
      <title
        id="title"
      >
        Добавить точку в маршрут
      </title>
      <path
        className="icon__path"
        d="M13.893 8.555v5.274H8.619a1.172 1.172 0 100 2.344h5.274v5.33a1.172 1.172 0 102.343 0v-5.33h5.274a1.172 1.172 0 100-2.344h-5.274V8.555a1.172 1.172 0 10-2.343 0zm14.935.879c.647 0 1.172.524 1.172 1.171v14.707A4.693 4.693 0 0125.312 30H4.688A4.693 4.693 0 010 25.312V4.688A4.693 4.693 0 014.688 0h20.625A4.693 4.693 0 0130 4.688a1.172 1.172 0 11-2.344 0 2.346 2.346 0 00-2.343-2.344H4.688a2.346 2.346 0 00-2.344 2.344v20.625a2.346 2.346 0 002.344 2.343h20.625a2.346 2.346 0 002.343-2.343V10.605c0-.648.525-1.172 1.172-1.172z"
        fill="#E9E9E9"
      />
    </svg>
  </button>
  <p
    className="new-point__description"
  >
    Не определено место положения
  </p>
</div>
`;
